src/components/Button.tsx

```diff
@@ -0,0 +1,20 @@
+export const Button = ({
+  classes,
+  label,
+  handleUpdate,
+}: {
+  classes: string;
+  label: string;
+  handleUpdate: () => {};
+}) => {
```

Comment:
Avoid declaring types inside params. Use Props instead. Also explicitly declare the React.FC type. 
`export const Button: React.FC<Props> = ({ ... }) => { ... }`